db/index.js

```diff
@@ -34,6 +29,27 @@ mySqlConnection.connect((error) => {
 
 const populator = function () {
   for (let i = 0; i < 100; i + 1) {
+    username = faker.name.findName();
+    userAddress = faker.address.streetAddress();
+    userCity = faker.address.city();
+    userZip = faker.address.zipCode();
+    userState = faker.address.state();
+    userPrime = faker.random.boolean();
+    userlistName = faker.random.word();
+    itemName = faker.commerce.productName();
+    itemDesc = faker.lorem.sentence();
+    itemPriceNew = faker.commerce.price();
+    itemPriceUsed = faker.commerce.price();
+    itemStock = faker.random.number();
```

Comment:
Why not just create and assign these variables here as opposed to on lines 4 - 15? Do we need these anywhere else after the populator function?

Reply:
Good idea, I'll make those changes!